### ISSUE TITLE
Defold Struggling on Linux

### ISSUE THREAD

**Issue by SpencerWF** [defold/editor2-issues#2857]
### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2019-10-24T16:19:34.173423</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>be024b63994ffb88d462db9275b6d9f9e553c5d6</td></tr>
<tr><td>Defold engine sha</td><td>13261949f45c333806c8aac8bd5b08124ca2810f</td></tr>
<tr><td>Defold version</td><td>1.2.163</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"0750f411d281427cb5184c95fac385bb"'>0750f411d281427cb5184c95fac385bb</a></td></tr>
<tr><td>GPU</td><td>Mesa DRI Intel(R) HD Graphics 530 (Skylake GT2) </td></tr>
<tr><td>GPU Driver</td><td>3.0 Mesa 19.0.8</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Linux</td></tr>
<tr><td>OS version</td><td>4.15.0-72-generic</td></tr>
<table>

**Comment by vlaaad** [comment]
Is there a file at `~/.Defold/unpack/be024b63994ffb88d462db9275b6d9f9e553c5d6/x86_64-linux/lib/libPVRTexLib.so`? Can you share the latest logs (Available at Help → Show Logs)?

**Comment by bagucode** [comment]
I just tested with a fresh download of Defold on Arch Linux and it starts and builds a fresh project just fine. In addition to what Vlad asked for, can you also tell us what Linux distribution you are running and what you were doing?

**Comment by SpencerWF** [comment]
Yes, there is a libPVRTexLib.so file, do you want it alone or do you also want other logs?

Linux Distribution:
Ubuntu 18.04.3 LTS

Description:
Running Defold on dual-booted Ubuntu distro (defold is running fine).
Open Piskel via command line (Piskel a simple pixelart editor) edit some sprites and tilemaps for a while (30 minutes, Defold is running in the background). Switch back to Defold, it is very slow, non-responsive and after a short time crashes (sometimes I just close and reopen it once it's unresponsive).  First time reopening it after unresponsive state it freezes after opening a project, need to restart it a second time for it to return to normalcy.

I don't know if Piskel has anything to do with it, just seemed like it could.

[libPVRTexLib.zip](https://github.com/defold/editor2-issues/files/3943601/libPVRTexLib.zip)
